Content.Client/GameTicking/Managers/ClientGameTicker.cs

```diff
@@ -155,11 +154,34 @@ private void RoundEnd(RoundEndMessageEvent message)
             RestartSound = message.RestartSound;
 
             // Don't open duplicate windows (mainly for replays).
-            if (_window?.RoundId == message.RoundId)
+            if (RoundEndSummaryWindow?.RoundId == message.RoundId)
                 return;
 
             //This is not ideal at all, but I don't see an immediately better fit anywhere else.
-            _window = new RoundEndSummaryWindow(message.GamemodeTitle, message.RoundEndText, message.RoundDuration, message.RoundId, message.AllPlayersEndInfo, _entityManager);
+            RoundEndSummaryWindow = new RoundEndSummaryWindow(message.GamemodeTitle, message.RoundEndText, message.RoundDuration, message.RoundId, message.AllPlayersEndInfo, _entityManager);
+        }
+    }
+
+    [UsedImplicitly, AnyCommand]
+    public sealed class ChangelogCommand : IConsoleCommand
+    {
+        public string Command => "roundendsummary";
+        public string Description => "Opens the round end summary window";
+        public string Help => $"Usage: {Command}";
+
+        public void Execute(IConsoleShell shell, string argStr, string[] args)
+        {
+            var entitySystem = IoCManager.Resolve<IEntitySystemManager>();
+            var clientGameTicker = entitySystem.GetEntitySystem<ClientGameTicker>();
+            var window = clientGameTicker.RoundEndSummaryWindow;
+            if (window == null)
+            {
+                shell.WriteLine(Loc.GetString("round-end-summary-command-no-window"));
+                return;
+            }
+
+            window.OpenCenteredRight();
+            window.MoveToFront();
         }
     }
```

Comment:
Needs localising + dependencies not IOC resolves, also shouldn't be useable by anyone should be behind debug.

Reply:
why would reopening roundend summary require perms

also command is named Changelog pro

Reply:
> is named Changelog

Ya, I copy-pasted it from the changelog command. Same goes for the openahelp one. They are all the same.

Reply:
> Needs dependencies not IOC resolves

I'd argue that it makes no difference in that particular situation. Decorating it and allowing a factory to resolve it is O(1) that involves some reflections shenanigans, and calling IoC resolve, which is backed by a dictionary, also is O(1). Not to mention that it's client-side.

---

Anyway, I'm adding the entity system as a decorated dependency. Can't resolve the ticker right away, as it's unregistered.

> Unhandled exception. Robust.Shared.IoC.Exceptions.UnregisteredDependencyException: Content.Client.GameTicking.Managers.RoundSummaryCommand requested unregistered type with its field Content.Client.GameTicking.Managers.ClientGameTicker: _game